apps/browser-proxy/src/tls.ts

```diff
@@ -31,3 +34,12 @@ export async function getTls() {
     key: Buffer.from(key),
   }
 }
+
+// cache the TLS certificate for 1 week
+const cache = new ExpiryMap(1000 * 60 * 60 * 24 * 7)
```

Comment:
What if this server restarted 1 day before renewal? I suppose the old cert is still good for another week, so this should be safe?

Reply:
I need to check but I think we renew the certificate monthly while the validity is 3 months. So in any cases with 2 week refresh we should be safe.